app/lib/forms/have_ofsted_urn.rb

```diff
@@ -20,7 +20,11 @@ def next_step
     end
 
     def previous_step
-      :qualified_teacher_check
+      if wizard.store["works_in_nursery"] == "yes"
```

Comment:
one for next time, I added `Services::QueryStore` to try to contain the arbitrary store calls